buildspec/packageTestVsix.yml

```diff
@@ -1,4 +1,4 @@
-version: 0.2
+version: 0.3
```

Comment:
What is this?

Reply:
It doesn't seem to match the CodeBuild versions

https://docs.aws.amazon.com/codebuild/latest/userguide/build-spec-ref.html#build-spec-ref-versions

Reply:
I was under the assumption that was the script version? Makes sense, will change back.